job-dsl-core/src/main/docs/examples/javaposse/jobdsl/dsl/helpers/publisher/PublisherContext/deployToWeblogic.groovy

```diff
@@ -0,0 +1,43 @@
+job('example') {
+    publishers {
+        deployToWeblogic {
+            // these are the default values used for optional fields, when not overridden by closure
+            mustExitOnFailure false
```

Comment:
Add the round brackets for method calls. I know they are optional, but all examples use them and I want the examples to be consistent.